extensions/desktop-common/desktop-common

```diff
@@ -58,8 +58,8 @@ function is_subpath() {
 
 if [ "$WITH_SNAP_DESKTOP_RUNTIME" = "yes" ]; then
   # add general paths not added by snapcraft due to runtime snap
-  append_dir LD_LIBRARY_PATH "$SNAP_DESKTOP_RUNTIME/lib/$SNAP_LAUNCHER_ARCH_TRIPLET"
-  append_dir LD_LIBRARY_PATH "$SNAP_DESKTOP_RUNTIME/usr/lib/$SNAP_LAUNCHER_ARCH_TRIPLET"
+  append_dir LD_LIBRARY_PATH "$SNAP_DESKTOP_RUNTIME/lib/$SNAP_DESKTOP_ARCH_TRIPLET"
```

Comment:
And when this directory is not added to `LD_LIBRARY_PATH`, pixbuf loaders can't find `libgdk_pixbuf-2.0.so.0` when it's in a content provider such as `kde-frameworks-5-core18`.

Reply:
does the fix in 6c46d4b7 make this work correctly for the KDE runtime?

Reply:
Yes, issue fixed, thanks.